ibis_substrait/compiler/translate.py

```diff
@@ -441,6 +441,17 @@ def _translate_window_bounds(
     return translate_preceding(*preceding), translate_following(*following)
 
 
+@translate.register(ops.Alias)
```

Comment:
Should we do this in a backwards compatible way? E.g., importing `Alias` and only registering if the import succeeds?

Reply:
hmm, it's a good question.  I'm not sure what, if any, backwards compatibility guarantees we're prepared to make at this point.  We could make it backwards compatible (at least for now) just to demonstrate that we can, I suppose.  
I realize I'm not even certain if `ibis-substrait` is compatible with ibis 2.1.1

Reply:
Roger that.  I'll add in some version gating